main/streams/plain_wrapper.c

```diff
@@ -40,6 +40,7 @@
 # include "win32/time.h"
 # include "win32/ioutil.h"
 # include "win32/readdir.h"
+#include <limits.h>
```

Comment:
Nit: `# include` for consistency.

Reply:
if you look higher up, you'll see several of these inconsistencies in the previous 40 lines.